recipe_roulette.py

```diff
@@ -69,9 +71,9 @@ def get_random_recipe():
     ingredient = get_user_input()
     try_counter = 0
     while True:
-        sanitize_ingredient_for_url = ingredient.replace(' ', '+')
-        url = 'http://www.recipepuppy.com/api/?i={}&p={}'.format(sanitize_ingredient_for_url, random.randint(1,100))
-        request_recipe = requests.get(url)
+        sanitized_ingredient_for_url = ingredient.replace(' ', '+') # using verbs to name variables is bad practice 
+        url = 'http://www.recipepuppy.com/api/?i={}&p={}'.format(sanitized_ingredient_for_url, random.randint(1,100))
+        request_recipe = requests.get(url) # url doesn't need to be a var if this is the only place it's going to be evoked

```

Comment:
Does this comment suggest editing to: request_recipe = requests.get( 'http://www.recipepuppy.com/api/?i={}&p={}'.format(sanitized_ingredient_for_url, random.randint(1,100))), rather than generating a variable for the url, which does not seem to save space because the variable is not referenced again?

Reply:
Yes, though I didn't change it because I figured might have preferred using a variable to splitting the method call to multiple lines and/or breaking pep8